cypress/support/step_definition/common_helper/globalActionSteps.spec.js

```diff
@@ -0,0 +1,22 @@
+const { When, Then } = require("@badeball/cypress-cucumber-preprocessor");
```

Comment:
- common_helper folder is for pre & post condition steps.

- globalAssertion steps can be put inside the commonRoomBoardSteps.js  inside the step definition of the room_board folder.

Reply:
no, these steps are global, not only for Roomboard, so I think, this is the right place in common helper.

Reply:
the table we used will be used also for other parts in the SVS